### ISSUE TITLE
NaT in date columns make histograme go crazy

### ISSUE THREAD

**Issue by mbh86** [pandas-profiling/pandas-profiling#507]
Hello,

I am referencing issue 111, which happens again in `version 2.8.0`

```
import pandas as pd
import pandas_profiling
import numpy as np


def test_issue_111():
    years = pd.Series(["{:02d}/01/{}".format(x, y) for x in range(1989, 2020) for y in range(1, 30)])

    df = pd.DataFrame(np.random.randint(0, len(years), size=(len(years), 4)), columns=list('ABCD'))
    df['E'] = pd.to_datetime(years)
    df['E'].iloc[10:100] = pd.NaT

    report = df.profile_report()
    report.to_file('test.html')
```

_From the post by @sbrugman in https://github.com/pandas-profiling/pandas-profiling/issues/111#issuecomment-497034063_

**Comment by sbrugman** [comment]
Hi @mbh86, thanks for opening a new issue. I'm looking into this for the next release. 

In the meantime: could you add a screencapture with the current histogram and the expected result?

**Comment by mbh86** [comment]
Hi @sbrugman,

here is the current result:

![image](https://user-images.githubusercontent.com/10547700/86766959-8b473000-c04b-11ea-9e26-1456b0fabb8f.png)

And the expected result is: (which I obtain when I filter out missing values in the dataframe)

![image](https://user-images.githubusercontent.com/10547700/86768567-16c1c080-c04e-11ea-8fcb-e16a4086a49e.png)

**Comment by sbrugman** [comment]
I've worked on precomputing histograms in parallel. This included your bug report in the refactor, and it should be fixed in the next release.

**Comment by sbrugman** [comment]
The v2.9.0rc1 release is out, and should resolve this issue.
Until this version is fully released, you can install it via pip in the following way:
```
pip install --pre -U pandas-profiling
```

It would be very helpful to know if the release candidate adequately solves the issue.

**Comment by mbh86** [comment]
@sbrugman, I tested it and it works, thank you.
